core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -515,6 +511,10 @@ public PropServerConfiguration(
             this.rerunInitialWaitQueueSize = getIntSize(properties, env, "http.busy.retry.initialWaitQueueSize", 64);
             this.rerunMaxProcessingQueueSize = getIntSize(properties, env, "http.busy.retry.maxProcessingQueueSize", 4096);
 
+            this.circuitBreakerThrottle = getInt(properties, env, "circuit.breaker.throttle", 2_000_000);
+            this.circuitBreakerBufferSize = getInt(properties, env, "circuit.breaker.buffer.size", 64);
+            this.circuitBreakerMaxTime = getInt(properties, env, "circuit.breaker.max.time", 60) * Timestamps.SECOND_MICROS;
```

Comment:
Everything is microseconds this one is seconds, would be good to hint it in the name. Also this is quite fundamental parameter so I'd rename key for this one to `query.max.timeout.secs`

Reply:
fixed here: https://github.com/questdb/questdb/pull/1637/commits/c1a9425b8640e63d443748b2edd64abe0c5cd398